packages/smart-contracts/package.json

```
@@ -40,6 +40,7 @@
   },
   "devDependencies": {
     "@openzeppelin/contracts": "2.4.0",
+    "ethers": "4.0.40",
```

```suggestion
    "ethers": "4.0.38",
```
let's be consistent in the versions of dependencies we use